modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-common-terms_{context}"]
+= Glossary of common terms for OpenShift {gitops-shortname}
+
+This glossary defines common OpenShift {gitops-shortname} terms.
+
+Argo CD::
+An open-source declarative tool that automates the continuous deployment (CD) of Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+{gitops-shortname}::
+{gitops-shortname} is a declarative way to implement continuous deployment for cloud native applications. In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state.
+
+{gitops-title}::
+ An Operator that uses Argo CD as the declarative {gitops-shortname} engine for enabling {gitops-shortname} workflows across multicluster OpenShift and Kubernetes infrastructure.
+
+{gitops-shortname} CLI ({gitops-shortname} `argocd` CLI)::
+A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+
+Control plane ({gitops-shortname} control plane)::
+In the {gitops-shortname} context, control plane is the `openshift-gitops` namespace, where, by default, the {gitops-title} installs an instance of Argo CD. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. Examples include a control plane repository wherein you can create one or more application custom resources and express an intent to deploy and sync Kubernetes manifests from Git.
++
+Within the `openshift-gitops` namespace, Argo CD maintains a set of following Kubernetes resources, which define the continuous deployment between the source Git repository and destination clusters:
```

Comment:
## 1)

The Argo CD control plane is any namespace that Argo CD is installed, rather than just the openshift-gitops namespace. 

Since openshift-gitops IS the default, that is also the default control plane, but if a user decides to install their own non-default Argo CD in a different namespace, then the control plane is different (not openshift-gitops).

I think we should tweak the definition's references to openshift-gitops to reflect that it is the default, but other namespaces can be control plane.

## 2)

I think this sentence can be removed: `Examples include a control plane repository wherein you can create one or more application custom resources and express an intent to deploy and sync Kubernetes manifests from Git.`

IMHO this is outside the definition of GitOps control plane (or, perhaps, is a secondary definition).

Reply:
> Since openshift-gitops IS the default, that is also the default control plane, but if a user decides to install their own non-default Argo CD in a different namespace, then the control plane is different (not openshift-gitops).

@jgwest Please elaborate on this. 

- Are you saying that users can change this default control plane namespace? So if I want to have `Sriabc` as the namespace to install Argo CD by default, then is that the new control plane for me?
- Also, what do you mean by non-default ArgoCD? If that is the user-defined control plane then why is it non-default? I am confused.

Reply:
Revised definition, thanks to @anandf's assistance in helping me understand this concept better:

In the {gitops-shortname} context, you can have a control plane for every Argo CD you install. A {gitops-shortname} control plane is any namespace, where, you can install an Argo CD along with its Argo CD instance. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. 
+
Within a control plane namespace, Argo CD maintains a set of following Kubernetes resources, which define the continuous deployment between the source Git repository and destination clusters:

* Argo CD `Application` custom resources (CRs)
* `ConfigMap` API objects
* `Secret` objects representing the {gitops-shortname} repository credentials and cluster credentials for deployment targets

By default, {gitops-title} installs an instance of Argo CD in the `openshift-gitops` control plane namespace with additional permissions for managing certain cluster-scoped resources. The {gitops-shortname} Operator manages this `openshift-gitops` namespace.